en/advanced/rtk_gps.md

```diff
@@ -1,54 +1,60 @@
 
-The datalink should typically be able to handle an uplink rate of 300 bytes per
-second. See the Uplink Datarate section below for more.
+The datalink should typically be able to handle an uplink rate of 300 bytes per second (see the [Uplink Datarate](#uplink-datarate) section below for more information).
 
-### Automatic Configuration
+## Supported RTK GPS modules
 
-Both QGroundControl and the autopilot firmware share the same [PX4 GPS driver stack](https://github.com/PX4/GpsDrivers). In practice, this means that support for new protocols and/or messages only need to be added to one place.
+PX4 currently only supports the single-frequency (L1) u-blox M8P based GNSS receivers for RTK. 
 
-The PX4 GPS stack automatically sets up the u-blox M8P modules to send and receive the correct messages over the UART or USB, depending on where the module is connected (to QGroundControl or the autopilot). No configuration using U-Center is necessary.
+A number of manufacturers have created products using this receiver. The list of devices that we have tested can be found [in the user guide](https://docs.px4.io/en/advanced_features/rtk-gps.html#supported-rtk-devices).
 
-As soon as the autopilot receives `GPS_RTCM_DATA` mavlink messages, it
-automatically forwards the RTCM data to the attached GPS module.
+> **Note** u-blox has two variants of the M8P chip, the M8P-0 and the M8P-2. 
+  The M8P-0 can only be used as Rover, not as Base, whereas the M8P-2 can be used both as Rover or as Base.
 
-> **Note** u-blox has two variants of the M8P chip, the M8P-0 and the M8P-2. The
-> M8P-0 can only be used as Rover, not as Base, whereas the M8P-2 can be used
-> both as Rover or as Base.
 
-#### RTCM Messages
+## Automatic Configuration
+
+The PX4 GPS stack automatically sets up the u-blox M8P modules to send and receive the correct messages over the UART or USB, depending on where the module is connected (to *QGroundControl* or the autopilot). 
+
+As soon as the autopilot receives `GPS_RTCM_DATA` mavlink messages, it automatically forwards the RTCM data to the attached GPS module.
+
+> **Note** The U-Center RTK module configuration tool is not needed/used!
+
+<span></span>
+> **Note** Both *QGroundControl* and the autopilot firmware share the same [PX4 GPS driver stack](https://github.com/PX4/GpsDrivers). In practice, this means that support for new protocols and/or messages only need to be added to one place.
+
+
+### RTCM messages
 
 QGroundControl configures the RTK base station to output the following RTCM3.2 frames, each with 1 Hz:
+
 - **1005** - Station coordinates XYZ for antenna reference point (Base position).
 - **1077** - Full GPS pseudo-ranges, carrier phases, Doppler and signal strength (high resolution).
 - **1087** - Full GLONASS pseudo-ranges, carrier phases, Doppler and signal strength (high resolution).
 
 
-### Uplink Datarate
+## Uplink datarate
```

Comment:
I could improve this but I need to clarify MAVLink2 vs MAVLink1. 

This says "The length of each MAVLink message is 182 bytes," -  Does the 182 bytes have any relevance to MAVLink2 as well - ie is it a "maximum" length or is this purely a mavlink1 thing.

My interpretation as it is written now is that this is a maximum size for both MAVLink1 and 2. However on MAVLInk 1 the message is always sent at this size while Mavlink2 can send a message that varies in size based on the content.